README.md

```diff
@@ -1,24 +1,23 @@
 # App Bridge Authentication
 
-> __Note:__ This project uses resources that are currently in a public beta
-
-A demo app created using Rails, React, and App Bridge for the Shopify tutorial [Build a Shopify app with Rails, React, and App Bridge][1].
+A demo app created using Rails, React, and App Bridge for the Shopify tutorial [Build a Shopify app with Rails, React, and App Bridge](https://shopify.dev/tutorials/build-rails-react-app-that-uses-app-bridge-authentication).
 
 ## Quick Start
 
 To run this app locally, you can clone this repository and do the following.
 
-1. Create a `.env` file to specify your `Shopify API Key` and `Shopify API Secret` available from your partners dashboard.
+1. Create a `.env` file to specify this app's `API key` and `API secret key` app credentials that can be found in the Shopify Partners dashboard.
 
 ```
-SHOPIFY_API_KEY='YOUR API KEY FROM SHOPIFY PARTNERS DASHBOARD'
-SHOPIFY_API_SECRET='YOUR API SECRET KEY FROM SHOPIFY PARTNERS DASHBOARD'
+SHOPIFY_API_KEY=<The API key app credential specified in the Shopify Partners dashboard>
+SHOPIFY_API_SECRET=<The API secret key app credential specified in the Shopify Partners dashboard>
+APP_URL=<The app URL specified in the Shopify Partners dashboard>
 ```
 
-> __Note:__ If you do not have a Shopify API Key or Shopify API Secret, see the following sections of the [Build a Shopify App with Node and React][2] guide:
-> 1. [Expose your dev environment][3]
-> 2. [Get a Shopify API Key and Shopify API secret key][4]
-> 3. [Add the Shopify API Key and Shopify API secret key][5]
+> __Note:__ If you do not have a Shopify API Key or Shopify API Secret, see the following sections of the [Build a Shopify App with Node and React](https://shopify.dev/tutorials/build-a-shopify-app-with-node-and-react/embed-your-app-in-shopify#get-a-shopify-api-key) guide:
+> 1. [Expose your dev environment](https://shopify.dev/tutorials/build-a-shopify-app-with-node-and-react/embed-your-app-in-shopify#expose-your-dev-environment)
+> 2. [Get a Shopify API Key and Shopify API secret key](https://shopify.dev/tutorials/build-a-shopify-app-with-node-and-react/embed-your-app-in-shopify#get-a-shopify-api-key)
+> 3. [Add the Shopify API Key and Shopify API secret key](https://shopify.dev/tutorials/build-a-shopify-app-with-node-and-react/embed-your-app-in-shopify#add-the-shopify-api-key)
```

Comment:
`SHOPIFY_API_SECRET` vs `SHOPIFY_API_SECRET_KEY` in this link.

Reply:
Added a disclaimer.